interface/ceed-basis.c

```diff
@@ -429,27 +396,28 @@ int CeedBasisGetFlopsEstimate(CeedBasis basis, CeedTransposeMode t_mode, CeedEva
         break;
     }
   } else {
-    CeedInt dim, num_comp, num_nodes, num_qpts, Q_comp;
+    CeedInt dim, num_comp, num_nodes, num_qpts;
     CeedCall(CeedBasisGetDimension(basis, &dim));
     CeedCall(CeedBasisGetNumComponents(basis, &num_comp));
     CeedCall(CeedBasisGetNumNodes(basis, &num_nodes));
     CeedCall(CeedBasisGetNumQuadraturePoints(basis, &num_qpts));
-    CeedCall(CeedBasisGetNumQuadratureComponents(basis, &Q_comp));
+    CeedFESpace fe_space;
+    CeedCall(CeedBasisGetFESpace(basis, &fe_space));
     switch (eval_mode) {
       case CEED_EVAL_NONE:
         *flops = 0;
         break;
       case CEED_EVAL_INTERP:
-        *flops = num_nodes * num_qpts * num_comp;
+        *flops = num_nodes * num_qpts * num_comp * (fe_space == CEED_FE_SPACE_H1 ? 1 : dim);
```

Comment:
It'd be nice to keep these more generic calculations if we can

Reply:
Resolved in [063bea1](https://github.com/CEED/libCEED/pull/1156/commits/063bea108b882e83a5ae894d5a49e974101cbd30).